reference/configuration/doctrine.rst

```diff
@@ -492,42 +492,35 @@ you can control. The following configuration options exist for a mapping:
 type
 ....
 
-One of ``annotation``, ``xml``, ``yml``, ``php`` or ``staticphp``. This
-specifies which type of metadata type your mapping uses.
+One of ``annotation`` (the default value), ``xml``, ``yml``, ``php`` or
+``staticphp``. This specifies which type of metadata type your mapping uses.
 
 dir
 ...
 
-Path to the mapping or entity files (depending on the driver). If this path
-is relative it is assumed to be relative to the bundle root. This only works
-if the name of your mapping is a bundle name. If you want to use this option
-to specify absolute paths you should prefix the path with the kernel parameters
-that exist in the DIC (for example ``%kernel.project_dir%``).
+Absolute path to the mapping or entity files (depending on the driver). The
+default value is ``%kernel.project_dir%/src/Entity/``.
 
 prefix
 ......
 
-A common namespace prefix that all entities of this mapping share. This
-prefix should never conflict with prefixes of other defined mappings otherwise
-some of your entities cannot be found by Doctrine. This option defaults
-to the bundle namespace + ``Entity``, for example for an application bundle
-called AcmeHelloBundle prefix would be ``Acme\HelloBundle\Entity``.
+A common namespace prefix that all entities of this mapping share. This prefix
+should never conflict with prefixes of other defined mappings otherwise some of
+your entities cannot be found by Doctrine. This option defaults to
+``App\Entity`` because it's recommended to store the entities in ``src/Entity/``.
 
 alias
 .....
 
 Doctrine offers a way to alias entity namespaces to simpler, shorter names
-to be used in DQL queries or for Repository access. When using a bundle
-the alias defaults to the bundle name.
+to be used in DQL queries or for Repository access. It's default value is ``App``.
```

Comment:
Btw, I took out these "default value is". These aren't really the default values - it's just what you get out-of-the-box with the recipe. And I think if you're configuring these things, then you are clearly looking at your `doctrine.yaml` file, so you will see those values there I think not mentioning these is a bit less confusing.